src/SubmitPage/SubmitForm.js

```diff
@@ -161,7 +161,12 @@ class SubmitForm extends React.Component {
     return "";
   }
 
+  canBeSubmitted() {
```

Comment:
* code looks good. from an experience point of view, it's not clear what will enable the button. i would suggest keeping the button enabled throughout, and show a validation error message near the submit button.

* we would also need the logic to work on open/close time fields, since they have been added to the form now (was not the case when the issue was first created)

* so if at least one of open, close, useful, safety is filled in => good to go

* if all fields are empty, we should prompt them to fill in the useful information field, since that is the most flexible.

Reply:
Great, Thanks @sampoorna @arjun27 , Yup, that makes sense. Yup, I was wondering how we were gonna do a UX around this. So we we do a form Validation like (times || useful info || safety info) and if that fails, we prompt a message?

Reply:
yeah that sounds good. you might be able to use the alert component: https://react-bootstrap.github.io/components/alerts/